resolwe_bio/docker_images/dnaseq/packages-manual/r-packages.sh

```diff
@@ -2,9 +2,18 @@
 
 . /var/cache/build/packages-manual/common.sh
 
+# These steps need to be performed because certain packages produce a "database corrupt error". The solution is to
+# restart R, but this is not feasible during the docker built or building of dependencies. This chunk also installs
+# packages from bioconductor. This entire step should be considered temporary until we can get packrat to work for
+# packages from all universes (CRAN, Bioconductor).
+Rscript --slave --no-save --no-restore-history -e "install.packages('assertthat')"
```

Comment:
All three `Rscript` commands can be joined not to repeat the first line.

Reply:
My reasoning for leaving them in three chunks is that they belong to three conceptual groups. The first one is a hack, that emulates restarting of R after installing the offending `assertthat` package. Second chunk is the explicit installation of the Bioconductor packages, which are a dependency in `chemut` package. And finally, the last chunk installs the package that we really want.
Of course I could always run things in one/two chunks and place comments in there, but since it's just an aesthetic thing and there's no performance hit, I would leave it the way it is. If you really insist on this, I can merge the chunks.

Reply:
You can leave it as it is. It was just an idea.